src/app/admin/question/new-question/new-question.component.ts

```diff
@@ -0,0 +1,167 @@
+import { Component, OnInit } from '@angular/core';
+import { FormGroup, FormControl, FormBuilder, Validators, ValidatorFn, AbstractControl } from '@angular/forms';
+import { ActivatedRoute, Router } from '@angular/router';
+import { QuestionService } from '../question.service';
+import { switchMap } from 'rxjs/operators';
+import { of } from 'rxjs';
+import { IAnswer } from '../question';
+
+@Component({
+  selector: 'app-new-question',
+  templateUrl: './new-question.component.html',
+  styleUrls: ['./new-question.component.scss']
+})
+export class NewQuestionComponent implements OnInit {
+
+  constructor(
+    private route: ActivatedRoute,
+    private router: Router,
+    private questionService: QuestionService
+  ) { }
+
+  newQuestionForm = new FormGroup({
+    question_text: new FormControl(''),
+    level: new FormControl('1'),
+    attachment: new FormControl(''),
+    type: new FormControl('1'),
+  }, {validators: this.questionTaskAdded()});
```

Comment:
Why didn't you use required validator for `question_text` and `attachment`?

Reply:
Seems like question can have only attachment added (no question text) or only question text (no attachment). I will clarify this with Yurii.